src/platyPS/platyPS.psm1

```diff
@@ -1416,7 +1416,7 @@ function GetMarkdownFilesFromPath
             }
             elseif (Test-Path -PathType Container $_)
             {
-                $MarkdownFiles += Get-ChildItem $_ -Filter $filter | WHERE {$_.BaseName -notlike $aboutFilePrefixPattern}
+                $MarkdownFiles += Get-ChildItem $_ -Filter $filter | Where-Object {$_.BaseName -notlike $aboutFilePrefixPattern}
```

Comment:
Nice. Completely agree with expanding out aliases. Make it easier to understand for more people.

Reply:
VS Code makes it no-brainer with automatic alias expansion refactoring.
I seen a bunch of folders like `-ErrorAction` in repo root, but hopefully fixed all `mkdir` calls.